styles.css

```diff
@@ -258,26 +258,35 @@ body {
     text-align: justify;
 }
 
-.collapsible {
+#manga-description-div {
     position: relative;
     max-height: 3.6em;
     overflow: hidden;
-    transition: max-height 0.5s ease;
+    transition: max-height 0.5s linear;
+}
+
+#manga-description {
+    margin-bottom: 1.8em;
 }
 
 .fade-out {
     position: absolute;
-    top: 1.8em;
+    height: 1.8em;
     bottom: 0;
     left: 0;
     right: 0;
     background: linear-gradient(transparent, rgba(51, 51, 51, 1));
-    transition: opacity 0.5s ease;
     cursor: pointer;
 }
 
-.expanded {
-    max-height: 1000em;
+#description-expand-icon {
+    font-size: 2.7em;
+    margin: 0px;
+    transition: transform 0.5s linear;
+}
+
+#manga-description-div.expanded {
+    max-height: 50em;
```

Comment:
Capped this at 50em because large values (like 1000) were causing issues with transition. Doesn't allow full description if it gets longer than `(font-size * 50)px`. Might need to do some testing and increase the value or implement it differently.

Reply:
Fixed in [934b78a](https://github.com/Animeboynz/Mihon-Backup-Viewer/pull/23/commits/934b78ac537a1ae4229b0e0bca575e8d9d3422cb).